frontend/util.lua

```diff
@@ -564,4 +564,18 @@ function util.htmlToPlainTextIfHtml(text)
     return text
 end
 
+-- Encode the HTML entities in a string.
+-- @string text the string to escape
+-- Taken from https://github.com/kernelsauce/turbo/blob/master/turbo/escape.lua
```

Comment:
I'd ignore any potential line length concerns to simply include the full link like https://github.com/kernelsauce/turbo/blob/e4a35c2e3fb63f07464f8f8e17252bea3a029685/turbo/escape.lua#L58-L70

I mainly mean the commit hash; the line numbers are just a convenience.

Reply:
I've updated it to the full link.